### ISSUE TITLE
DeprecationWarning for uuid@7 package

### ISSUE THREAD

**Issue by SeWaS** [nestjs/nest#4211]
## Bug Report

With yesterday's nestJS release (6.11.9) it seems that deprecation warnings because of unwanted ways to import the `uuid` package are thrown. 

## Current behavior

When running my application or tests for my application, it seems that nestJS logs a deprecation warning because of deprecated imports of the `uuid@7` package:
```bash
(node:63608) DeprecationWarning: Deep requiring like `const uuidv4 = require('uuid/v4');` is deprecated as of uuid@7.x. Please require the top-level module when using the Node.js CommonJS module or use ECMAScript Modules when bundling for the browser. See https://github.com/uuidjs/uuid/blob/master/README.md#upgrading-from-v3x-of-this-module for more information.
```
It seems that these warnings are printed by the nestJS framework. When running `npm run test` in the nest-repo, we can see this warnings as well.
## Input Code

Imports like 

```ts
import * as uuid from 'uuid/v4';
```
like done in `injectable.decorator.ts`, for instance, seem to be deprecated with verion7 of uuid (https://github.com/uuidjs/uuid/blob/master/README.md#upgrading-from-uuid3).

## Expected behavior

No deprecation warnings 😃
## Possible Solution

Switch import statements from 
```ts
import * as uuid from 'uuid/v4';
```
to
```ts
import { v4 as uuidv4 } from 'uuid';
```
as mentioned in the `uuid` documentation.
## Environment

<pre><code>
Nest version: 6.11.9 

 
For Tooling issues:
- Node version: 13.8.0
- Platform:  Mac

Others:

</code></pre>

**Comment by kamilmysliwiec** [comment]
Fixed in 6.11.10. Thanks for reporting

**Comment by SeWaS** [comment]
Perfect. Working like a charm!

**Comment by Upsetmania** [comment]
nest v6.14.2
nodejs: v13.10.1
the message comes back when I launch graphql forRootAsync
npm run start: dev

**Comment by Upsetmania** [comment]
ok thanks fix version 7.0.0 ;)

**Comment by Gougerik** [comment]
> Fixed in 6.11.10. Thanks for reporting

no matching version found for uuid@6.11.10

**Comment by Ivanov-Mihail** [comment]
@SeWaS , @kamilmysliwiec thanks guys for solution!
